packages/opera.rb

```diff
@@ -3,29 +3,34 @@
 class Opera < Package
   description "Opera isn't just a browser. It's an awesome browser."
   homepage 'https://www.opera.com/'
-  version '71.0.3770.198'
+  version '71.0.3770.271'
   compatibility 'x86_64'
   case ARCH
   when 'x86_64'
     source_url 'file:///dev/null'
     source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
     depends_on 'alien' => :build
     depends_on 'gtk3'
+    depends_on 'gsettings_desktop_schemas'
     depends_on 'sommelier'
   end
 
   binary_url ({
-    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/opera-71.0.3770.198-chromeos-x86_64.tar.xz',
+    x86_64: 'http://download1525.mediafire.com/txdclyv3qd0g/ix1ewo821h6d14n/opera-71.0.3770.271-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    x86_64: '826b9da7012782f422196509033f7ba0068bd7d885e0984d709d222bdbe1718d',
+    x86_64: 'ee4449b99e5afe6ead245a5bb487fd7ec22552bf7a5076bc686d0eee758e1518',
   })
 
   def self.build
     system "wget https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera-stable_#{version}_amd64.deb") ) == 'd44af29613a8eb00ea54e63a74ce48d601131b239e191c695cc6124bb746a00f'
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera-stable_#{version}_amd64.deb") ) == 'cad630af33ce7d54de802abd9c360b19d330ef64759888da1ec03e5dd7e54f9a'
+    system "sudo chmod -R 777 * "
     system "alien -tc opera-stable_#{version}_amd64.deb"
+    system "sudo chmod -R 777 * "
+    system "chmod -R +x *"
     system "tar xvf opera-stable-#{version}.tgz"
+    system "sudo chmod -R 777 * "
```

Comment:
Is this necessary?  I've never needed to change perms since the chronos user has write access to `/usr/local` already.  Or are you doing this just to get rid of the alien warning?  That warning is no biggie.

Reply:
I want to fix the tar error with it to change the permissions but it don't works